src/main/java/org/folio/circulation/services/feefine/FeeRefundProcessor.java

```diff
@@ -134,7 +138,11 @@ private DateTime getActionDate(AccountRefundContext context) {
   }
 
   private String getTransferRefundTransactionInfo(Account account) {
```

Comment:
Is this really refund information now that it can be a refund or a credit?

Reply:
I think yes, there is `getTransferCreditTransactionInfo` for the credit action.